src/main/java/org/opensrp/web/rest/ClientResource.java

```diff
@@ -269,10 +292,44 @@ private int getTotal(ClientSearchBean clientSearchBean, AddressSearchBean addres
 			total = clientService.findTotalCountHouseholdByCriteria(clientSearchBean, addressSearchBean).getTotalCount();
 		} else if (pageNumber == FIRST_PAGE && clientType.equalsIgnoreCase(ALLCLIENTS)) {
 			total = clientService.findTotalCountAllClientsByCriteria(clientSearchBean, addressSearchBean).getTotalCount();
+		} else if (pageNumber == FIRST_PAGE && clientType.equalsIgnoreCase(ANC)) {
+			clientSearchBean.setClientType(null);
+			total = clientService.findCountANCByCriteria(clientSearchBean, addressSearchBean);
+		} else if (pageNumber == FIRST_PAGE && clientType.equalsIgnoreCase(CHILD)) {
+			clientSearchBean.setClientType(null);
+			total = clientService.findCountChildByCriteria(clientSearchBean, addressSearchBean);

```

Comment:
For this block could it be refactored to avoid the repetition of **pageNumber == FIRST_PAGE** . We could have the outer if with with check instead

Reply:
I have refactored  with outer if check, Please check